samples/msal-react-samples/onPageLoad/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/msal-react": "1.0.0-alpha.0",
-    "@azure/msal-browser": "2.7.0",
+    "@azure/msal-react": "file:../../../lib/msal-react",
```

Comment:
You shouldn't need to do this, lerna can take care of bootstrapping?

Reply:
Also beachball bumps the dependencies

Reply:
We removed samples from lerna to prevent things like puppeteer from being installed on every build

Reply:
Sorry I think I reviewed the PRs in the wrong order.